### PR TITLE
Use the shared eslint config

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: vue/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "browserslist-config-nextcloud": "0.0.1",
     "css-loader": "^3.2.0",
     "eslint": "^5.16.0",
+    "eslint-config-nextcloud": "0.0.4",
```
Comment (skjnldsv) [inline vue/package.json]: latest is 0.0.5 :thinking:

File: vue/src/Views/TurnServers.vue [hunk 2]
```diff
@@ -58,7 +60,7 @@
 <script>
 import { Tooltip } from 'nextcloud-vue'
 import debounce from 'debounce'
-import TurnServer from 'Components/TurnServer'
+import TurnServer from '../Components/TurnServer'
```
Comment (skjnldsv) [inline vue/src/Views/TurnServers.vue]: You can still use your custom aliases if you want, you just ened to keep this part of the config on your eslint, it's just a matter of personnal preference :)

Comment (nickvergessen) [reply vue/src/Views/TurnServers.vue]: :man_shrugging: dont care too much, so lets go with the "standard"

**Pull request merged**